src/NUnitFramework/framework/Api/FrameworkController.cs

```diff
@@ -481,6 +481,22 @@ private static void AddDictionaryEntries(TNode settingNode, IDictionary entries)
         /// </summary>
         public abstract class FrameworkControllerAction : LongLivedMarshalByRefObject
         {
+            /// <summary>
+            /// Wrap in NUnitCallContext executes the action within an NUnitCallContext
+            /// which ensures System.Runtime.Remoting.Messaging.CallContext is cleaned up
+            /// suitably at the end of the test run. This method only has an effect running
+            /// the full .NET Framework.
+            /// </summary>
+            /// <param name="action"></param>
```

Comment:
❔ Can you remove the `<param>` xml and use cref for `NUnitCallContext`? Maybe the summary could be started, `Executes`? Maybe `the` after `ensures`, and cref instead of `System.Runtime.Remoting.Messaging.CallContext`?

Reply:
Sure - good shout. 😊